krun/platform.py

```diff
@@ -647,6 +648,49 @@ def check_preliminaries(self):
         self._check_perf_samplerate()
         self._check_tickless_kernel()
         self._check_aslr_disabled()
+        self._check_realtime_throttle_disabled()
+
+    def _check_realtime_throttle_disabled(self, first=True):
```

Comment:
I didn't guess what `first` was for until I got to the end of this method. Can we call it something a bit more verbose, like `first_attempt` or `first_check`?